src/Checker/OrderRefundingAvailabilityChecker.php

```diff
@@ -27,7 +27,8 @@ public function __invoke(string $orderNumber): bool
 
         return in_array(
             $order->getPaymentState(),
-            [OrderPaymentStates::STATE_PAID, OrderPaymentStates::STATE_PARTIALLY_REFUNDED]
-        );
+            [OrderPaymentStates::STATE_PAID, OrderPaymentStates::STATE_PARTIALLY_REFUNDED]) &&
```

Comment:
Something went wrong with the formatting
```suggestion
            [OrderPaymentStates::STATE_PAID, OrderPaymentStates::STATE_PARTIALLY_REFUNDED] &&
```

Whole condition could look like this:
```php
        return in_array($order->getPaymentState(), [
            OrderPaymentStates::STATE_PAID, 
            OrderPaymentStates::STATE_PARTIALLY_REFUNDED,
        ] ) && $order->getTotal() !== 0;
```
or something similar